bluemira/geometry/tools.py

```diff
@@ -222,11 +222,68 @@ def make_polygon(
     -------
     wire: BluemiraWire
         a bluemira wire that contains the polygon
+
+    Notes
+    -----
+    If the input points are closed, but closed is True, the returned BluemiraWire will be
+    closed.
```

Comment:
Should this read '_If the input points are closed, but closed is **False**_, ...'?

Reply:
Also, do we have a test that verifies this?